util/compliance.py

```diff
@@ -0,0 +1,84 @@
+# This util searches a standards document against our procedure docs and returns likely
+# matches for each standard
+# Run with Python3!!
+
+
+from json import load
+from os import path, rename, mkdir, listdir
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize
+from nltk import download
+from whoosh.index import create_in, open_dir
+from whoosh.fields import Schema, TEXT
+from whoosh.qparser import QueryParser
+from whoosh import scoring
+
+directory = '/Users/jpdimarz/Repos/security-policy-templates/templates/procedures' #set this to the appropriate path
```

Comment:
Could this be made an argument, defaulting to something relative to the project? For example:

```sh
$ git clone git@github.com:JupiterOne/security-policy-templates.git
$ cd security-policy-templates
$ python3 util/compliance.py
```

Otherwise, we'd have to edit this file to run this file, and then revert the change to the working directly clean for any commits.

Reply:
Good point. I changed it to use argparse and added a README for documentation.